src/main/java/com/acme/dbo/txlog/ByteCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.acme.dbo.txlog;
+
+import static java.lang.Math.abs;
+
+public class ByteCommand {
+        private static String DECOR = "primitive: ";
+
+        public Byte currentValue;
+        private static String accumulator;
+        private static Byte sum;
+        private static Byte MAX_BYTE = Byte.MAX_VALUE;
+
+        public ByteCommand(Byte message) {
+            currentValue = message;
+            if (accumulator == null) {
+                accumulator = message.toString();
+                sum = message;
+            }
+        }
+
+        public static String getDecoratedState() {
+            return DECOR + accumulator;
+        }
+
+        public void accumulate() {
+            if (accumulator == null) {
+                accumulator = this.currentValue.toString();
+                sum = this.currentValue;
+            } else {
+                if (checkByteValueIsOutBound((byte) (this.currentValue))) {
+                    Controller.flush();
+                    accumulator = MAX_BYTE + "";
+                    sum = MAX_BYTE;
+                    Controller.flush();
+                } else {
+                    sum = (byte) (sum + this.currentValue);
+                    if (accumulator.contains(System.lineSeparator())) {
```

Comment:
3-level if and we need to go deeper ))) Please no ) Feel free to Extract methods, it won't hurt )

Reply:
Ready